torch/_inductor/codegen/cpp_wrapper_gpu.py

```diff
@@ -251,7 +255,29 @@ def generate_user_defined_triton_kernel(
         )
 
     def generate_tma_descriptor(self, desc):
```

Comment:
Because AOTI still uses two-pass run at the moment, does the cpp wrapper codegen here need to read any information from the python run? I have added things like `DeferredGpuGridLine` for that purpose. Do you think it is necessary to do something like that here?

Reply:
I believe, the Python and C++ codegens for the TMA descriptor creation are independent.